frontend/packages/pipelines-plugin/src/components/repository/RepositoryLinkList.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { GithubIcon } from '@patternfly/react-icons';
+import { useTranslation } from 'react-i18next';
+import { ResourceLink, ExternalLink, CopyToClipboard } from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { RepositoryModel } from '../../models';
+import { PipelineRunKind } from '../../types';
+import { baseURL, RepositoryLabels, RepositoryFields } from './consts';
+
+export type RepositoryLinkListProps = {
+  pipelineRun: PipelineRunKind;
+};
+
+const RepositoryLinkList: React.FC<RepositoryLinkListProps> = ({ pipelineRun }) => {
+  const { t } = useTranslation();
+  const plrLabels = pipelineRun.metadata.labels;
+  const repoLabel = RepositoryLabels[RepositoryFields.REPOSITORY];
+  const repoName = plrLabels && plrLabels[repoLabel];
+  const urlOrg = plrLabels[RepositoryLabels[RepositoryFields.URL_ORG]];
+  const urlRepo = plrLabels[RepositoryLabels[RepositoryFields.URL_REPO]];
+  const repoURL = urlOrg && urlRepo && `${baseURL}/${urlOrg}/${urlRepo}`;
+
+  if (!repoName) return null;
+
+  return (
+    <dl>
+      <dt>{t('pipelines-plugin~Repository')}</dt>
+      <dd>
+        <ResourceLink
+          kind={referenceForModel(RepositoryModel)}
+          name={repoName}
+          namespace={pipelineRun.metadata.namespace}
+        />
+        {repoURL && (
+          <ExternalLink href={repoURL}>
+            <GithubIcon title={repoURL} /> {repoURL}
+          </ExternalLink>
+        )}
+      </dd>
+      {plrLabels[RepositoryLabels[RepositoryFields.BRANCH]] && (
+        <>
+          <dt>{t('pipelines-plugin~Branch')}</dt>
+          <dd>{plrLabels[RepositoryLabels[RepositoryFields.BRANCH]]}</dd>
+        </>
+      )}
+      {plrLabels[RepositoryLabels[RepositoryFields.SHA]] && (
+        <>
+          <dt>{t('pipelines-plugin~SHA')}</dt>
```

Comment:
I think we want this to be `Commit Id` cc @bgliwa01 @karthikjeeyar 

SHA is the type of format the commit id is in, it's more useful to reference it as commit id as THAT's what it is. SHA is a format and can represent anything (not necessarily a commit id)